### PR TITLE
Milestone#4: Desktop version, added the breakpoint at 768px for desktop version. 

### PR DESCRIPTION
**Pull request by erickhansx**
Implemented the fourth part of the project for desktop:

- Implemented the breakpoint at 768px for the main section. 
- Implemented the breakpoint at 768px for the works section. 
- Implemented the breakpoint at 768px for the contact-me section. 
- Added several padding and margins to the section to look better on the desktop version.
- Adapted the font-sizing and weight to the desktop version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,250 +1,360 @@
...
+              </ul>
+              <ul class="li">
+                <li><a href="#">About</a></li>
+              </ul>
+              <ul class="li">
+                <li><a href="#">Contact</a></li>
+              </ul>
+            </nav>
+          </div>
+        </nav>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that your hamburger button comes in the middle of the top of your portfolio and that is responsive like the Figma design shows so you can give the `div` that holds the `header__nav-menu` `nav` a display none when you are in the mobile size.

Your Design | Figma Design 
---------------- | ------------------
Screenshot from 2022-06-01 22-42-27 | Screenshot from 2022-06-01 22-49-25

File: index.html [hunk 2]
```diff
@@ -1,250 +1,360 @@
...
+          <li class="css">
+            <img
+              class="ellipse-image"
+              src="/resources/img/css-ellipse.png"
+              alt="CSS Ellipse"
+            />CSS
           </li>
         </ul>
-        <button class="btn-2">Get in touch</button>
-      </form>
```
Comment (ShahierNashaat) [inline index.html:244]: - [ ] Kindly be noticed that the form should be wider in the desktop size to match the Figma design.

Your Design | Figma Design
---------------- | -------------------
Screenshot from 2022-06-01 22-45-17 | Screenshot from 2022-06-01 22-45-02

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @erickhansx , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Hi @erickhansx , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**